lesson07/assignment/assignment07/src/App.vue

```diff
@@ -0,0 +1,85 @@
+<template>
+  <div id="app">
+    <a href="#main-content" class="hidden">Skip to main content</a>
+    <div class="container-fluid">
+      <div id="nav">
+        <div class="row">
+           <!-- JUMBOTRON AND NAV -->
+          <header class="col-sm-12 col-12 jumbotron jumbotron-fluid background-image">
+            <nav class="navbar navbar-expand-lg d-flex flex-row justify-content-between align-items-start">
+              <div class="navbar-brand text-white text-uppercase d-flex flex-column align-items-center">
+                <h1 class="display-1">Ten Ton</h1>
+                <h1 class="display-3">Coffee</h1>
+              </div>
```

Comment:
The text in the header isn't quite as responsive as we'd like. When it fails, the text is nicely clipped, doesn't overflow into surrounding areas, which is good. I'd recommend further manual testing of responsivity here, and updating styling as necessary. 
<img width="931" alt="Screen Shot 2021-03-06 at 11 10 56 AM" src="https://user-images.githubusercontent.com/2523812/110218157-f3654480-7e6c-11eb-9eb8-8ce011993c7a.png">